packages/js/src/core/user-info.ts

```diff
@@ -11,7 +9,7 @@ type Identity = {
 type OrganizationData = {
   id: string;
   name: string;
-  description: Nullable<string>;
+  description?: string;
```

Comment:
Should we use `Nullable` from `@logto/essentials` instead? It sesms we were using a wrong import source

Reply:
@wangsijie